cronjobparser.py

```diff
@@ -134,23 +133,24 @@ class CronJobParser(object):
     
     def __init__(self, infile="/etc/crontab"):
         self.file = infile
+        self.content = ""
+        self.assignments = {}
+        self.cronjobs = []
         self.read()
 
     def read(self):
         f = codecs.open(self.file, "r", "utf-8")
         self.content = f.read()
         f.close()
         data = self.get()
-        self.assignments = {}
```

Comment:
I think it's OK to add the initialization of these members to ``__init__``, but they need to be reset to empty values in ``read`` as well. Otherwise, we would end up with duplicate entries in ``self.cronjobs`` becuase ``read`` may be called multiple times for the same ``CronJobParser`` object. :)

Reply:
Fixed in 1c0899a
Removed reading the file twice, in `read()` and in `get()`. Also added a test for re-reading a modified file.